doc/ref_model/chapters/chapter09.md

```diff
@@ -192,7 +192,24 @@ As such the requirements for Capacity Management on the infrastructure are cover
 
 <a name="9.5.2.2"></a>
 #### 9.5.2.2 Software CI/CD Requirements
-
+The requirements including for CI/CD for ensuring software security scans, image integrity checks, OS version checks, etc. prior to deployment, are listed in the Table XX.XX (below). Please note that the tenant processes for application LCM (such as updates) are out of scope. For the purpose of these requirements, CI includes Continuous Delivery, and CD refers to Continuous Deployment.
+
+Ref # | Description | Comments/Notes
+---|---|---
+auto.cicd.001 | The CI/CD pipeline must support deployment on any cloud and cloud infrastructures including different hardware accelerators. | CI/CD pipelines automate CI/CD best practices into repeatable workflows for integrating code and configurations into builds, testing builds including validation against design and operator specific criteria, and delivery of the product onto a runtime environment.<br>Example of an open-source cloud native CI/CD framework is the Tekton project (https://tekton.dev/)
+auto.cicd.002 | The CI/CD pipelines must use event-driven task automation | 
+auto.cicd.003 | The CI/CD pipelines should avoid scheduling tasks | 
+auto.cicd.004 | The CI/CD pipeline is triggered by a new or updated software release being loaded into a repository | The software release cane be source code files, configuration files, images, manifests.<br>Operators may support a single or multiple repositories and may, thus, specify which repository is to be used for these release.<br>An example, of an open source repository is the CNCF Harbor (https://goharbor.io/)
```

Comment:
@karinesevilla suggested making clear that images and manifests are also stored in the repository. 
```suggestion
auto.cicd.004 | The CI/CD pipeline is triggered by a new or updated software release being loaded into a repository | The software release can be source code files, configuration files, images, manifests, and these are all stored in a repository.<br>Operators may support a single or multiple repositories and may, thus, specify which repository is to be used for the release.<br>An example, of an open source repository is the CNCF Harbor (https://goharbor.io/)
```

Reply:
@pgoyal01 Good,I agree